pages/k8s/how-to-cos-lite.md

```diff
@@ -0,0 +1,149 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Integrating COS Lite with Charmed Kubernetes"
+  description: Integrating COS Lite with Charmed Kubernetes
+keywords: Cloud, cluster, observability
+tags: [observability, operating]
+sidebar: k8smain-sidebar
+permalink: how-to-cos-lite.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+**Charmed Kubernetes** includes the standard Kubernetes dashboard for
+monitoring your cluster. However, it is often advisable to have a monitoring
+solution which will run whether the cluster itself is running or not. It may
+also be useful to integrate monitoring into existing setups.
+
+To make monitoring your cluster a delightful experience, Canonical provides
+first-class integration between charmed-kubernetes and COS Lite. This guide
+will help you integrate a COS Lite deployment with a charmed-kubernetes deployment.
+
+This document assumes you have a controller with an installation of
+charmed-kubernetes. If this is not your case, refer to
+["how-to-install"](<https://ubuntu.com/kubernetes/docs/how-to-install>).
+
+## Preparing a platform for COS Lite
+
+First, create a microk8s model to act as a deployment cloud for COS Lite:
+
+```
+juju add-model --config logging-config='<root>=DEBUG' \
```

Comment:
We can maybe add a little side note on the command: a juju model is similar to a kubernetes namespace and we are setting the logging config to debug so that you can look at logs using juju debug-log.